core/client/get_and_verify.go

```diff
@@ -28,11 +28,12 @@ import (
 func (c *Client) VerifiedGetUser(ctx context.Context, userID string) (*types.MapRootV1, *pb.MapLeaf, error) {
 	c.trustedLock.Lock()
 	defer c.trustedLock.Unlock()
-	resp, err := c.cli.GetUser(ctx, &pb.GetUserRequest{
+	req := &pb.GetUserRequest{
 		DirectoryId:          c.DirectoryID,
 		UserId:               userID,
 		LastVerifiedTreeSize: int64(c.trusted.TreeSize),
-	})
+	}
+	resp, err := c.cli.GetUser(ctx, req)
```

Comment:
The change in this file is a no-op. Did you intend to use the new pair verifier below?

Reply:
I would like to, but I can't do that just yet because of the `updateTrusted` call. 
A few PRs later I might be able to